test/RachisTests/DatabaseCluster/ClusterDatabaseMaintenance.cs

```diff
@@ -822,7 +823,23 @@ public async Task RemoveNodeFromClusterWhileDeletion()
 
                 var node = Servers[1].ServerStore.Engine.Tag;
                 DisposeServerAndWaitForFinishOfDisposal(Servers[1]);
-                var res = await leaderStore.Maintenance.Server.SendAsync(new DeleteDatabasesOperation(databaseName, true));
+
+                Assert.ThrowsAsync<RavenException>(async () =>
```

Comment:
fix this

Reply:
![image](https://github.com/ravendb/ravendb/assets/6377808/ea14000b-d9f4-4e70-bd8a-22f3b99fa356)

Reply:
Done